src/data-type.js

```diff
@@ -1305,10 +1305,30 @@ const TYPE = module.exports.TYPE = {
       if (value instanceof Date) {
         return value;
       }
-      value = Date.parse(value);
-      if (isNaN(value)) {
+      var dateValue = Date.parse(value);
+      if (!isNaN(dateValue)) {
+        return dateValue;
+      }
+      var timespan = (function(input) {
```

Comment:
No, there is not specific reason :sweat_smile: I just wrote it and exists as you see. But I can change that ;)